Danielshow/Fast_Food_React
# #162053211 implement menu page

#### What does this PR do?
This PR implement menu page where a user can see a list of food to order from

#### Description of Task to be completed?
- implement menu page UI where users can see all menus
- fetch food from API
- all users if authenticated or not can see menu

#### How should this be manually tested?
- clone this branch `git clone ft-user-see-order-page-162053211`
- `yarn install` to install dependencies
- `yarn start` to start the application
- `yarn test` to run test

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162053211](https://www.pivotaltracker.com/story/show/162053211)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
